styles/components/label.scss

```diff
@@ -19,6 +19,61 @@
 //   font-size: #{$local-font-size}em;
 // }
 
+.title {
```

Comment:
This is specific to the docs, you can move it to `docs/styles/main.scss` or just leave it alone (aka, delete changes related to doc styling).